recipes/py_entitymatching/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "py_entitymatching" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "a8a70ecfe7488a7d1d48f15f0b24c4e7965bc31a8096aa65e2ad9337afcc6dfa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [not x86_64]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - msinttypes  # [vc9]
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - pandas
```

Comment:
THis is a lot of host dependencies.  Do all of them need to be here?

Reply:
We have removed some of the unnecessary host dependencies. Thanks for the help.